library/src/main/java/com/getstream/sdk/chat/ChatImpl.java

```diff
@@ -109,8 +118,46 @@ public String getVersion() {
         return BuildConfig.BUILD_TYPE + ":" + BuildConfig.VERSION_NAME;
     }
 
-    void init() {
+    @Override
+    public void setUser(@NotNull User user,
+                        @NotNull String userToken,
+                        @NotNull InitConnectionListener callbacks) {
```

Comment:
`InitConnectionListener` might be nullable. Developers can get connection event or error via `ChatObservable`.